Header header

uint8 battery
uint8 mechanic                # mechTemperature
uint8 selfTest
